website/docs/feature-flag-tutorials/python/implementing-feature-flags.md

```diff
@@ -0,0 +1,346 @@
+
+[Flask Surveys Container App](https://github.com/pamelafox/flask-surveys-container-app) is an example Python application using [Flask](https://flask.palletsprojects.com/en/3.0.x/) and [SQLAlchemy](https://www.sqlalchemy.org/) to create and store surveys. Flask is a Python framework that provides out-of-the-box configurations to get the shell of a full-stack web application up and running, which includes [Jinja](https://jinja.palletsprojects.com/en/3.1.x/) for building web page HTML templates served using Python routing. This sample app is containerized, meaning you won’t have to rely on your system’s configurations of different libraries and tools used to allow the app to run successfully. It’s wrapped in a Docker container that will install and run the app in isolation.
+
+![A screenshot of the Flask Surveys Container App](/img/python-tutorial-survey-example.png)
+
+In this tutorial, you will learn how to set up and use Python feature flags with Unleash. We will use the Flask Surveys container app to implement the feature flag solution, which will roll out a feature that allows users to delete surveys they create.
+
+At the end of this tutorial, you will be able to turn on the feature flag and activate a route that will remove surveys from the database.
+
+Here are the steps we will cover in this tutorial:
+1. [Feature flag best practices for back-end applications](#1-unleash-best-practice-for-backend-apps)
+2. [Spin up a local flag provider](#2-install-a-local-feature-flag-provider)
+3. [Configure a feature flag](#3-create-and-configure-the-feature-flag)
+4. [Add Unleash to a Python Flask app](#4-add-unleash-to-a-python-app)
+5. [Toggle the database deletion route](#5-use-a-feature-flag-to-roll-out-a-delete-method)
+6. [Verify the toggle experience](#6-verify-the-toggle-experience)
+7. [Improve feature flag error handling](#7-improve-a-feature-flag-implementation-with-error-handling)
+
+
+## Prerequisites
+
+
+In this tutorial, you will need the following:
+- A web browser like Chrome or Firefox
+- Git
+- Docker
+- (Optional) a code editor like Visual Studio Code
+
+
+![An architectural diagram of our Python app using Unleash feature flags](/img/python-flask-unleash-architecture.png)
+
+
+This architecture diagram breaks down how the Python app works with Unleash to control feature flags.
+
+Your Python app is an **Evaluation Service**, which is meant to evaluate feature flags and pass evaluated results down to the SDK in the client application. In this context, the results will be passed to the Python SDK. 
```

Comment:
This doesn't make sense to me at a glance. I think the arch diagram is good but the description doesn't match and seems incorrect to me. 

The SDK is the Evaluation Service, it receives raw data about toggles from Unleash, evaluates them and returns that result to the host application.

(really like the name Evaluation Service btw! I use Flag Engine in my own internal model but we don't have a consistent name for this idea yet)

Reply:
I tried to base this off of what I could find in the docs. I don't think we have these terms defined anywhere but we are currently using them so this is what I came up with. I'd like to simplify it to two short sentences what evaluation service and feature flag control service is in this particular context.

Reply:
That'd be awesome! I'm happy to help walk you through the arch!

If we have docs that are suggesting this is the correct data flow, please point them to me, I think they're either misleading or incorrect!

Reply:
Agreed. 
 - **Unleash** is the **Evaluation Service** / **Flag Engine**
 - **Unleash Python SDK** is what will help you connect to Unleash from your Python app
 - ...and your Python app is whatever you want it to be 🙂